tests/test_chi_ssa_2.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import COMMISSION, PASSED
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.chi_ssa_2 import ChiSsa2Spider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "chi_ssa_2.html"),
+    url="http://belmontcentral.org/about-ssa-2/ssa2-meeting-minutes-audit/",
+)
+spider = ChiSsa2Spider()
+
+freezer = freeze_time("2019-07-18")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
+"""
```

Comment:
This can be removed

Reply:
Just to clarify, this was for the comment and not the call to freezer.stop() correct?

Reply:
Yep, just for the comment